source/slang/slang-ir-diff-jvp.cpp

```diff
@@ -9,373 +9,741 @@
 namespace Slang
 {
 
+template<typename P, typename D>
+struct Pair
```

Comment:
Let's not use generic names like `Pair` here. Consider names like `DiffInstPair` to be more specific.